packit/cli/builds/local_build.py

```diff
@@ -38,30 +68,39 @@
         "release_suffix is specified in the configuration."
     ),
 )
+@click.option(
+    "--srpm",
+    default=None,
+    type=click.Path(exists=True, dir_okay=False),
+    help=(
+        "Allows you to specify SRPM to use for the RPM build instead of the"
+        "default which is building from specfile."
```

Comment:
```suggestion
        "Allows you to specify SRPM to use for the RPM build instead of the "
        "default which is built from the specfile."
```